Product/pom.xml

```diff
@@ -1132,6 +1133,11 @@
         <version>1.4.1</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
Jason - I did not move anything to bottom... I have sent a screen shot of how i am adding new version... i am not writing any text..